gen-interop.sh

```diff
@@ -1,15 +1,13 @@
 #! /bin/bash
 
 # Configuration Data -- order matters in these arrays. A new entry requires an entry in ALL "ta_" arrays
-ta_tlas=("acapy" "afgo" "javascript" "dotnet" "findy" "aries-vcx")
-ta_names=("Aries Cloud Agent Python" "Aries Framework Go" "Aries Framework JavaScript" "Aries Framework .NET" "Findy Agent" "AriesVCX")
-ta_shortnames=("ACA-Py" "AF-Go" "AFJ" "AF-.NET" "Findy" "VCX")
-ta_scopes=("AIP 1, 2" "AIP 2" "AIP 1" "AIP 1" "AIP 1" "AIP 1" "AIP 1")
-ta_exceptions=("None" "None" "Revocation" "Proof Proposal" "Credential Exchange" "Revocation" "Proof Proposals, Public Dids, Revocations")
+ta_tlas=("acapy" "javascript" "findy" "aries-vcx")
```

Comment:
FIndy should be removed asd well. Confirmed that it was archived as of May 28, 2024.

Reply:
Findy removed.